.github/workflows/idp-login-request-generator.yml

```diff
@@ -34,13 +34,16 @@ jobs:
           - runner-id: 6
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
       - id: 'auth'
-        uses: 'google-github-actions/auth@v0'
+        name: 'Authenticate to Google Cloud'
+        uses: 'google-github-actions/auth@v1'
         with:
-          credentials_json: '${{ secrets.TEST_RUNNER_GOOGLE_TOKEN }}'
+          credentials_json: ${{ secrets.TEST_RUNNER_GOOGLE_TOKEN }}
+
       - name: 'Set up Cloud SDK'
-        uses: 'google-github-actions/setup-gcloud@v0'
+        uses: 'google-github-actions/setup-gcloud@v1'
```

Comment:
same here, also v2.1.0 (https://github.com/google-github-actions/setup-gcloud/releases), likely would want to do "@v2" to catch minor updates as they come unless you have a reason to got for that older v1?